bugsnag-plugin-android-performance-compose/src/main/java/com/bugsnag/android/performance/compose/LoadingIndicator.kt

```diff
@@ -0,0 +1,56 @@
+package com.bugsnag.android.performance.compose
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.ui.Modifier
+import com.bugsnag.android.performance.internal.BugsnagPerformanceInternals
+import com.bugsnag.android.performance.internal.SpanCategory
+
+private const val CONDITION_TIMEOUT = 100L
+
+/**
+ * A LoadingIndicator is an invisible composable that will block any ViewLoad span from ending
+ * until the LoadingIndicator has been disposed. This provides an easy and effective way to
+ * accurately measure the time-to-interactivity for any Activity, Fragment or composable (when
+ * using [MeasuredComposable]).
+ *
+ * A screen may include any number of LoadingIndicators, and the screen is considered to be
+ * "loading" as long as at least one remains active.
+ *
+ * Typically this composable can be wrap or embedded within the content of a loading placeholder.
```

Comment:
Is it better to say "indicator" or was placeholder deliberate?

```suggestion
 * Typically this composable can wrap or be embedded within the content of a loading indicator.
```

Reply:
"placeholder" was deliberate - though maybe not the best wording. I wanted to avoid "loading indicator" since that is the name of the composable, so we would be suggesting nesting them inside each other (which doesn't make much sense)

Reply:
```suggestion
 * Typically this composable can wrap or be embedded within the content of a loading placeholder.
```